core/src/main/java/tc/oc/pgm/spawns/states/Observing.java

```diff
@@ -72,22 +74,34 @@ public void enterState() {
     }
 
     if (reset) {
-      // Give basic observer items
-      ObserverToolFactory toolFactory = smm.getObserverToolFactory();
-      player.getInventory().setItem(0, toolFactory.getTeleportTool(bukkit));
 
-      if (toolFactory.canUseEditWand(bukkit)) {
-        player.getInventory().setItem(1, toolFactory.getEditWand(bukkit));
-      }
-
-      // Let other modules give observer items
-      player.getMatch().callEvent(new ObserverKitApplyEvent(player));
-
-      // Apply observer spawn kit, if there is one
-      spawn.applyKit(player);
+      // Give basic observer items.
+      // This task is delayed to ensure Bukkit
+      // has recieved the locale of the player before giving items with localized names
+      player
+          .getMatch()
+          .getExecutor(MatchScope.LOADED)
+          .schedule(
+              () -> {
+                ObserverToolFactory toolFactory = smm.getObserverToolFactory();
+                player.getInventory().setItem(0, toolFactory.getTeleportTool(bukkit));
+
+                if (toolFactory.canUseEditWand(bukkit)) {
+                  player.getInventory().setItem(1, toolFactory.getEditWand(bukkit));
+                }
+
+                // Let other modules give observer items
+                player.getMatch().callEvent(new ObserverKitApplyEvent(player));
+
+                // Apply observer spawn kit, if there is one
+                spawn.applyKit(player);
+
+                player.getBukkit().updateInventory();
+              },
+              500,
```

Comment:
This just adds a race condition based on time. Should figure out when the settings are actually sent and apply after that. Possibly just give the tools at the start and on locale change re-send them

Reply:
SportPaper does have a PlayerLocaleChangeEvent, but that doesn't work with trying to move away from using SportPaper. The only other way I can think of for doing something on locale change would be to add a ProtocolLib packet listener for PacketPlayInSettings, but that has its own problems.

Reply:
Checking for the locale being different than previous locale i guess is always an option, but it's not the best one

Reply:
Since we're moving away from SportPaper, we need some alternative. @cswhite2000 what's the issue with using a ProtocolLib packet listener?

Reply:
We would have to add ProtocolLib as a dependency for pgm, and it doesn't seem like its worth doing that for this. But, if we planning on using protocollib for supporting multiple versions for the tab list, then it probably would be worth it.

Reply:
Could do this as a soft-dependency then, where the functionality is disabled if ProtocolLib isn't on the server (similar to JGit handling).